tests/conftest.py

```diff
@@ -112,9 +113,21 @@ def docker_client() -> DockerClient:
 
 @fixture(scope="session")
 def gprofiler_docker_image(docker_client: DockerClient) -> Iterable[Image]:
-    image: Image = docker_client.images.build(path=str(PARENT))[0]
-    yield image
-    docker_client.images.remove(image.id, force=True)
+    # access the prebuilt image.
+    # this is built in the CI, in the "Build gProfiler image" step.
+    yield docker_client.images.get("gprofiler")
+
+
+@fixture(scope="session")
+def gprofiler_docker_image_resources(gprofiler_docker_image: Image):
+    """
+    Copy all resources from the built image.
+    """
+    copy_file_from_image(
+        gprofiler_docker_image,
+        os.path.join("/app", "gprofiler", "resources"),
+        os.path.join(os.path.dirname(gprofiler.__file__)),
+    )
```

Comment:
delete the files after the tests cycle finish

Reply:
nah, why